components/constellation/constellation.rs

```diff
@@ -263,6 +266,7 @@ pub struct Constellation<Message, LTF, STF> {
     pending_changes: Vec<SessionHistoryChange>,
 
     /// The currently focused pipeline for key events.
+    /// FIXME(paul): there should be one per top level browsing context.
```

Comment:
I'll remove the comment. I believe this is necessary. This can be discussed here: https://github.com/servo/servo/issues/17401